FrontEnd/src/components/terms-and-conditions-app/terms_conditions/TermsAndConditionsComponent.jsx

```diff
@@ -1,38 +1,35 @@
-import './Text.css';
-import './TermsAndConditionsComponent.css';
+import styles from './TermsAndConditionsComponent.module.css';
+import TermsAndConditionsText from './text';
+import Text from './text';
+import { v4 as uuidv4 } from 'uuid';
 import { useEffect } from 'react';
-import { termsConditions } from './Text';
-
-export function TermsAndConditions() {
-  useEffect(() => {
-    window.scrollTo(0, 0);
-  }, []);
+import LinkContainer from '../../CookiesPolicyPage/LinkContainer.jsx';
+import ContentRenderer from '../../CookiesPolicyPage/RenderingTextContainer.jsx';
 
+const TermsAndConditions = () => {
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, []);
   return (
-    <div className="block-terms_and_conditions block-size">
-      <div className="root-container">
-        <div className="divider" />
-        <div className="terms-conditions-main">
-          <div className="title-container">
-            <h2 className="title-terms_and_conditions">Terms & Conditions</h2>
-            <p className="description">{termsConditions.intro}</p>
-          </div>
+    <div className={styles['TermsAndConditions']}>
+        <div className={styles['TermsAndConditions__link_container']}>
+          <LinkContainer /> {}
+          <img className={styles['TermsAndConditions__img1']}
+                src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots.png`}
+                alt="dots.png" />
+          <img className={styles['TermsAndConditions__img2']}
+                src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots.png`}
+                alt="dots.png" />
+        </div>
+        <div className={styles['TermsAndConditions__text_container']}>
+        <h2 className={styles['TermsAndConditions__title']}>{TermsAndConditionsText.title} </h2>
+        {Text.content.map((item) => (
```

Comment:
Same could we use a more meaningful name?

Reply:
done